runtime/vara/src/lib.rs

```diff
@@ -1057,9 +1064,7 @@ pub struct DelegateFeeAccountBuilder;
 impl DelegateFee<RuntimeCall, AccountId> for DelegateFeeAccountBuilder {
     fn delegate_fee(call: &RuntimeCall, who: &AccountId) -> Option<AccountId> {
         match call {
-            RuntimeCall::GearVoucher(pallet_gear_voucher::Call::call { call }) => {
-                GearVoucher::sponsor_of(who, call)
-            }
+            RuntimeCall::GearVoucher(voucher_call) => voucher_call.sponsored_by(who.clone()),
```

Comment:
not related to this PR, but better to open a separate issue for the TODO over the `DelegateFee` trait impl.